02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/06 PyTorch/06 Predict Labels.html

```diff
@@ -0,0 +1,14 @@
+<p>To predict the labels of new data, in the OnData method, get the most recent set of features and then call the predict method.</p>
```

Comment:
OnData => `OnData`

Reply:
It says "call the predict method", but there is no `predict` method in the following snippet